src/common/cm/cm_test.cpp

```diff
@@ -507,17 +507,12 @@ CM_BoundsIntersect
 */
 bool CM_BoundsIntersect( const vec3_t mins, const vec3_t maxs, const vec3_t mins2, const vec3_t maxs2 )
 {
-	if ( maxs[ 0 ] < mins2[ 0 ] - SURFACE_CLIP_EPSILON ||
+	return ! ( maxs[ 0 ] < mins2[ 0 ] - SURFACE_CLIP_EPSILON ||
```

Comment:
negative conditions like this are impossible to read. We should remove the `!` and change all the || to &&

Reply:
Done.